### ISSUE TITLE
ACCESSIBILITY  MIRADOR – improve UX for download options  

### ISSUE THREAD

**Issue by brittnylapierre** [crkn-rcdr/demo-blacklight#11]
Supporting Comments  
- DEAF-BLIND – iPhone 13 Pro Max/iOS 17.2/Focus 14 Blue 4th gen/VoiceOver/Safari - Thought the document was a PDF after scrolling down and reading the download buttons. (Commented on misleading labeling, as the file should be identified as a PDF instead of just images) 
- DEAF-BLIND – iPhone 13 Pro Max/iOS 17.2/Focus 14 Blue 4th gen/VoiceOver/Safari - Suggested making download the PDF for alternative access a feature 
- DEAF-BLIND – Windows 10/JAWS 2019/Focus 14 braille/Chrome - Confusion with buttons and links for JPG and PDF downloads. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Successful JPG download but confusion with two buttons for PDF 
- LIMITED SIGHT – Apple Mac Mini M1 2000/MacOS 13.4.1/27” external monitor/Increased contrast /1280 by 72px/zooming - Download option located at the bottom of the page; suggested having download buttons in the top bar for easier access. Difficulty finding the download option; small text required zooming in; understanding the buttons after hovering over them 
- LIMITED SIGHT – Apple Mac Mini M1 2000/MacOS 13.4.1/27” external monitor/Increased contrast /1280 by 72px/zooming - Visibility issue distinguishing between JPG and PDF download options 
- LIMITED SIGHT – Apple Mac Mini M1 2000/MacOS 13.4.1/27” external monitor/Increased contrast /1280 by 72px/zooming - Difficulty finding download buttons in the bottom left corner. Suggested placement in the top main control section. 
- LIMITED SIGHT – Windows 10/NVDA 2022.4/ Chrome - Download button starting with the document size is confusing 
- LIMITED SIGHT – iPhone 14 Pro/iOS 17.2/VoiceOver/Safari - Pressing "open full-sized image in new tab" only labeled as "image" by VoiceOver. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Issues with the download area, particularly the full PDF download saving as a Chrome download instead of a PDF. Lack of clarity on how to open the downloaded PDF 
- DEAF – BLIND – Ipad Pro 9/iPadOS 17.2/Focus 40 Blue 4th gen/VoiceOver/Safari - It is unclear which page the link to open an image in a new tab will open
